cg/cli/clean.py

```diff
@@ -456,6 +456,7 @@ def remove_old_demultiplexed_run_dirs(context: CGConfig, days_old: int, dry_run:
         try:
             flow_cell: DemultiplexFlowCell = DemultiplexFlowCell(flow_cell_path=flow_cell_dir)
         except FlowCellError:
+            LOG.info(f"Skipping flow cell in {flow_cell_dir} due to FlowCellError")
```

Comment:
Maybe "except FlowCellError as error" and "Skipping flow cell in {flow_cell_dir} due to {error}" if it adds any value on top of stating that there is a FlowCellError? Feel free to keep as is if it is not worth the change, though.

Reply:
it is worth the change, thanks!

Reply:
fixed